.gitattributes

```diff
@@ -0,0 +1,2 @@
+# This is a generated file from vscode-l10n tool, and it always uses 'lf' and line ending. Set it to 'lf' so we don't get warning every time when localization file get changed.
+l10n/bundle.l10n.json text eol=lf
```

Comment:
Do we need to do this for the other files too?

Reply:
I only see this file shows up in the git warning. If you see anything else maybe add them in future?